packages/drizzle/src/utilities/buildIndexName.ts

```diff
@@ -0,0 +1,24 @@
+import type { DrizzleAdapter } from '../types.js'
+
+export const buildIndexName = ({
+  name,
+  adapter,
+  number = 0,
+}: {
+  adapter: DrizzleAdapter
+  name: string
+  number?: number
+}): string => {
+  const indexName = `${name}_idx${number ? `_${number}` : ''}`
```

Comment:
I would prefer to put the number in the middle than at the end: `${name}_${number}_idx`.